book-src/tour/tuples.md

```diff
@@ -4,7 +4,13 @@ Lists are good for when we want a collection of one type, but sometimes we want
 to combine multiple values of different types. In this case tuples are a quick
 and convenient option.
 
+Gleam provides two ways to construct or match on tuples: the `#(1, 2, 3)` format,
+introduced in Gleam 0.14.5, and the original `tuple(1, 2, 3)` format.
```

Comment:
Mind the release version, it's probably going to be 0.15.0

Reply:
I'll fix.